File: Vanilla/sources/zdoc/media/lua/shared/Library/ChannelCategory.lua
---@class ChannelCategory : zombie.radio.ChannelCategory
---@field public Undefined ChannelCategory
---@field public Radio ChannelCategory
---@field public Television ChannelCategory
---@field public Military ChannelCategory
---@field public Amateur ChannelCategory
---@field public Bandit ChannelCategory
---@field public Emergency ChannelCategory
---@field public Other ChannelCategory
ChannelCategory = {}

---@public
---@param arg0 String
---@return ChannelCategory
function ChannelCategory:valueOf(arg0) end

---@public
---@return ChannelCategory[]
function ChannelCategory:values() end

File: Vanilla/sources/zomboid/zombie/radio/ChannelCategory.java
package zombie.radio;

public enum ChannelCategory {
   Undefined,
   Radio,
   Television,
   Military,
   Amateur,
   Bandit,
   Emergency,
   Other;

   // $FF: synthetic method
   private static ChannelCategory[] $values() {
      return new ChannelCategory[]{Undefined, Radio, Television, Military, Amateur, Bandit, Emergency, Other};
   }
}
